doc/modules/learning_curve.rst

```diff
@@ -144,7 +144,7 @@ average scores on the validation sets)::
   >>> from sklearn.model_selection import learning_curve
   >>> from sklearn.svm import SVC
 
-  >>> train_sizes, train_scores, valid_scores = learning_curve(
+  >>> train_sizes, train_scores, valid_scores, _, _ = learning_curve(
```

Comment:
This should beak

Reply:
Thanks for the catch! Reverted back in f3fe244